codegenerator/cli/templates/static/codegen/src/ContractAddressingMap.res

```diff
@@ -23,7 +22,6 @@ let addAddress = (map: mapping, ~name: string, ~address: Address.t) => {
 
 /// This adds the address if it doesn't exist and returns a boolean to say if it already existed.
 let addAddressIfNotExists = (map: mapping, ~name: string, ~address: Address.t): bool => {
-  let address = address->Address.Evm.checksum
```

Comment:
Cool, I just want to double check that we sanitize the value passed into register contract function. Since the user could pass any string.

Reply:
I'll check